luthier/config/src/lib/json.lua

```diff
@@ -22,6 +22,9 @@
 -- SOFTWARE.
 --
 
+-- Modified by Gabriel on May 29, 2019 to create a better formatting when encoding 
+-- JSON. Slack: @gluo; GitHub: @gab-umich
```

Comment:
I wouldn't keep any of these changes. There's no point
in trying to make the JSON "pretty." In the normal course of things it's only consumed
by the computer, and if a human needs to look at it, there are a lot of tools available,
like `python -m json.tool` or `jq` to prettify it. Generating pretty JSON is unnecessary
complexity.

Reply:
I agree. Modularizing JSON formatting is vastly better.